Image-Processing/BFR_Sample_Rest.ipynb

```diff
@@ -0,0 +1,576 @@
+    "res = r.json()\n",
+    "print(json.dumps(res, indent=2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Create the indexer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "indexername = f'{datasource_container}-idxr'\n",
+    "indexer_def = {\n",
+    "    \"name\": f'{indexername}',\n",
+    "    \"description\": \"Indexer to enrich hotel reviews\",\n",
+    "    \"dataSourceName\": f'{datasource_container}-ds',\n",
+    "    \"skillsetName\": f'{datasource_container}-ss',\n",
+    "    \"targetIndexName\": f'{datasource_container}-idx',\n",
+    "    \"disabled\": None,\n",
+    "    \"schedule\": {\n",
+    "        \"interval\": \"PT2H\",\n",
+    "        \"startTime\": \"0001-01-01T00:00:00Z\"\n",
+    "      },\n",
+    "    \"parameters\": {\n",
+    "    \"batchSize\": None,\n",
+    "    \"maxFailedItems\": 0,\n",
+    "    \"maxFailedItemsPerBatch\": 0,\n",
+    "    \"base64EncodeKeys\": None,\n",
+    "    \"configuration\": {\n",
+    "      \"dataToExtract\": \"contentAndMetadata\",\n",
+    "      \"parsingMode\": \"default\",\n",
+    "      \"imageAction\": \"generateNormalizedImages\"\n",
+    "        }\n",
+    "    },\n",
+    "   \"fieldMappings\": [\n",
+    "    {\n",
+    "      \"sourceFieldName\": \"metadata_storage_path\",\n",
+    "      \"targetFieldName\": \"metadata_storage_path\",\n",
+    "      \"mappingFunction\": {\n",
+    "        \"name\": \"base64Encode\"\n",
+    "      }\n",
+    "    }\n",
+    "    ],\n",
+    "    \"outputFieldMappings\": [\n",
+    "        {\n",
+    "          \"sourceFieldName\": \"/document/normalized_images/*/text\",\n",
+    "          \"targetFieldName\": \"image_text\"\n",
+    "        }\n",
+    "    ],\n",
+    "    \"cache\": {\n",
+    "        \"enableReprocessing\": True,\n",
+    "        \"storageConnectionString\": f'{know_store_cache}'\n",
+    "    }\n",
+    "}\n",
+    "r = requests.post(construct_Url(search_service, \"indexers\", None, None, api_version), data=json.dumps(indexer_def), headers=headers)\n",
+    "print(r)\n",
+    "res = r.json()\n",
+    "print(json.dumps(res, indent=2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Run the indexer"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "r = requests.post(construct_Url(search_service, \"indexers\", indexername, \"run\", api_version), data=None,  headers=headers)\n",
+    "print(r)\n",
+    "#res = r.json()\n",
+    "#print(json.dumps(res, indent=2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### View Results\n",
+    "Once the indexer completes execution, you will find the image slices in the knowledge store. The following cell downloads a few images to validate the skill did work."
```

Comment:
line 518, last sentence doesn't read well.  How about "The following cell downloads a few images so that you can verify skillset success."